index.rst

```diff
@@ -104,7 +107,7 @@ Nighttime Operations
 Troubleshooting
 ======================
 
-All related to troubleshooting is included in this section. 
+This section features information about how to recover from faults during afternoon checkout or nighttime operations. If the problem persists after exhausting all documented procedures, ensure safety of the equipment, stop work, and call for help.
```

Comment:
So this section should be about unexpected faults that aren't covered elsewhere in the documentation.
Consider:
`This section includes information on current known issues that are not addressed in other areas of the documentation.
Content includes temporary issues and potential work-arounds. 
The section is continually updated as the observatory system evolves and develops.`

Reply:
👍